### PR TITLE
Python3

### PR DESCRIPTION
**Pull request by homm**
Two commits. First with general code style fixes

### CODE REVIEW

**Comment by homm** [comment]
@andrewshkovskii Review this

File: asyncdynamo/asyncdynamo.py [hunk 1]
```diff
@@ -180,72 +178,81 @@ def raise_error():
...
-        request.path = '/' # Important! set the path variable for signing by boto (<2.7). '/' is the path for all dynamodb requests
-        request.auth_path = '/' # Important! set the auth_path variable for signing by boto(>2.7). '/' is the path for all dynamodb requests
+        body = body.encode('utf-8')
+        headers = {'X-Amz-Target': '%s_%s.%s' % (self.ServiceName,
+                                                 self.Version, action),
+                   'Content-Type': 'application/x-amz-json-1.0',
+                   'Content-Length': str(len(body))}
+        request = HTTPRequest(self.url, method='POST', headers=headers,
+                              body=body, validate_cert=self.validate_cert)
+        request.path = '/'  # Important! set the path variable for signing by boto (<2.7). '/' is the path for all dynamodb requests
```
Comment (andrewshkovskii) [inline asyncdynamo/asyncdynamo.py:215]: Why didn't you made this and next comment on a new line?

**Review by andrewshkovskii — APPROVED** [review]
Looks good.
For next time it's better to split PR like this into two, because changes on docstrings are messed with code formatting and etc.

File: asyncdynamo/asyncdynamo.py (hunk 1 above)
Comment (homm) [reply asyncdynamo/asyncdynamo.py:215]: The aim was to minimize notifications in PyCharm rather than make the code actually better.

**Pull request merged**